### PR TITLE
Cities outside of U.S. can now return data

### PR DESCRIPTION
**Pull request by thaitwo**
Can you also look over this branch whenever you have some free time? 😄 🙏  Thank you!!!!

- Cities outside of the US can now retrieve weather data

### CODE REVIEW

File: dev/jsx/weather.jsx [hunk 1]
```diff
@@ -7,6 +7,10 @@ import SearchBar from './search-bar';
 import WeatherCard from './weather-card';
 import Geosuggest from 'react-geosuggest';
 
+const API_KEY = '5332856fca0fe1e7';
```
Comment (alexwolfe) [inline dev/jsx/weather.jsx]: You may want to create a `const.js` file that you import that has API key, URLs, etc for your app.

File: dev/jsx/weather.jsx [hunk 2]
```diff
@@ -21,32 +25,77 @@ class FetchWeather extends React.Component {
...
-        conditionsData: conditions.data.current_observation,
-        forecastData: forecast.data.forecast.simpleforecast.forecastday,
-        hourlyData: hourly.data.hourly_forecast,
-        dailyData: daily.data.forecast.simpleforecast.forecastday,
-        currentdate: date,
+    // Check to see if the country value is equal to 'usa'. If yes, then make Ajax call to get weather info
+    // This URL syntax is specific to cities only in the US
+    if (this.props.params.country === 'usa') {
+      axios.all([
+        axios.get(`${URL_BASE}/conditions/q/${this.props.params.state}/${this.props.params.city}.json`),
```
Comment (alexwolfe) [inline dev/jsx/weather.jsx:31]: 👍

**Review by alexwolfe — COMMENTED** [review]
Noice! done reviewing.

**Comment by thaitwo** [comment]
Alright, all conflicts have been resolved. Branch should be ready for merge. Thanks!

**Pull request merged**